src/bindings/capi/tvgCapi.cpp

```diff
@@ -555,6 +558,23 @@ TVG_EXPORT Tvg_Result tvg_scene_clear(Tvg_Paint* scene)
     return (Tvg_Result) reinterpret_cast<Scene*>(scene)->clear();
 }
 
+/************************************************************************/
+/* SVG Loader                                                           */
+/************************************************************************/
+TVG_EXPORT Tvg_Result tvg_svg_path_append_to_shape(Tvg_Paint* paint, const char* svgPath)
```

Comment:
could you please elaborate why can't we use the evas_vg__shape_append_svg_path from evas? 
I'm asking, since we had this discussion when we started the integration process. The conclusion was, that we will make the conversion on the evas site, and so we did.